solidity/contracts/test/TestQuerySender.sol

```diff
@@ -32,13 +33,13 @@ contract TestQuerySender {
         bytes calldata _targetData,
         uint256 _gasAmount
     ) external payable {
-        bytes32 _messageId = queryRouter.query(
+        queryAndPayFor(
```

Comment:
Make sure documented in drivebys if not otherwise

Reply:
this is just a test helper?